<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RPSLiSp</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="widgetStack">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="introPage">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="rightMargin">
         <number>1</number>
        </property>
        <item>
         <widget class="QLabel" name="logoImg">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>images/logo.png</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QTextEdit" name="rules">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Rock, Paper, Scissors, Lizard, Spock&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:700; font-style:italic;&quot;&gt;RULES&lt;/span&gt;:&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&amp;quot;&lt;span style=&quot; font-weight:700;&quot;&gt;Scissors&lt;/span&gt; cuts &lt;span style=&quot; font-weight:700;&quot;&gt;paper&lt;/span&gt;, &lt;span style=&quot; font-weight:700;&quot;&gt;paper&lt;/span&gt; covers &lt;span style=&quot; font-weight:700;&quot;&gt;rock&lt;/span&gt;, &lt;span style=&quot; font-weight:700;&quot;&gt;rock&lt;/span&gt; crushes &lt;span style=&quot; font-weight:700;&quot;&gt;lizard&lt;/span&gt;, &lt;span style=&quot; font-weight:700;&quot;&gt;lizard&lt;/span&gt; poisons &lt;span style=&quot; font-weight:700;&quot;&gt;Spock&lt;/span&gt;, &lt;span style=&quot; font-weight:700;&quot;&gt;Spock&lt;/span&gt; smashes &lt;span style=&quot; font-weight:700;&quot;&gt;scissors&lt;/span&gt;, &lt;span style=&quot; font-weight:700;&quot;&gt;scissors&lt;/span&gt; decapitates &lt;span style=&quot; font-weight:700;&quot;&gt;lizard&lt;/span&gt;, &lt;span style=&quot; font-weight:700;&quot;&gt;lizard&lt;/span&gt; eats &lt;span style=&quot; font-weight:700;&quot;&gt;paper&lt;/span&gt;, &lt;span style=&quot; font-weight:700;&quot;&gt;paper&lt;/span&gt; disproves &lt;span style=&quot; font-weight:700;&quot;&gt;Spock&lt;/span&gt;, &lt;span style=&quot; font-weight:700;&quot;&gt;Spock&lt;/span&gt; vaporizes &lt;span style=&quot; font-weight:700;&quot;&gt;rock&lt;/span&gt;, and as it always has, &lt;span style=&quot; font-weight:700;&quot;&gt;rock&lt;/span&gt; crushes &lt;span style=&quot; font-weight:700;&quot;&gt;scissors&lt;/span&gt;.&amp;quot; &lt;span style=&quot; font-style:italic;&quot;&gt;-Sheldon Cooper&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;AI:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;The computer will start to learn how you play and will try and counter your moves. There is also a &lt;span style=&quot; font-weight:700;&quot;&gt;3&lt;/span&gt; second window that you must play within or you lose to the computer by default. Good luck!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="playerName">
            <property name="placeholderText">
             <string>Enter your name...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="startGame">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Lets Play!</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="gamePage">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QProgressBar" name="gameTime">
            <property name="mouseTracking">
             <bool>false</bool>
            </property>
            <property name="value">
             <number>24</number>
            </property>
            <property name="textVisible">
             <bool>false</bool>
            </property>
            <property name="invertedAppearance">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="playerLabel">
                <property name="text">
                 <string>&lt;h1&gt;Player&lt;/h1&gt;</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLCDNumber" name="playerScore">
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="segmentStyle">
                 <enum>QLCDNumber::Filled</enum>
                </property>
                <property name="value" stdset="0">
                 <double>0.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Preferred</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="Line" name="line_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Preferred</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>&lt;h1&gt;Computer&lt;/h1&gt;</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLCDNumber" name="computerScore"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTextEdit" name="gameLog">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>15</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="btnRock">
              <property name="text">
               <string>Rock</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnPaper">
              <property name="text">
               <string>Paper</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnScissors">
              <property name="text">
               <string>Scissors</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnLizard">
              <property name="text">
               <string>Lizard</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnSpock">
              <property name="text">
               <string>Spock</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
